doc/admin-guide/plugins/multiplexer.en.rst

```diff
@@ -90,54 +91,3 @@ Here are some example :file:`remap.config` configuration lines::
 #. The fourth entry will multiplex requests sent to ``http://www.example.com`` with a path of ``/d``
    to both ``host1.example.com`` and ``host2.example.com``, but POST and PUT requests will
    not be multiplexed.
-
-Implementation
-==============
-
-Parsing Chunk Encoded Data
---------------------------
```

Comment:
I'm removing this because I don't think this will be helpful to end users and may confuse them into thinking that we support multiplexing chunk encoded requests. We do parse and send them onto the origin, but we do not support multiplexing them.